kairon/shared/chat/user_media.py

```
@@ -78,7 +78,8 @@ def save_media_content(
                 filename: str = None,
                 root_dir: str = None,
                 output_filename: str = None,
-                bucket: str = None
+                bucket: str = None,
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)